core/storage/story/gae_models_test.py

```diff
@@ -48,6 +48,36 @@ def test_get_deletion_policy(self) -> None:
             story_models.StoryModel.get_deletion_policy(),
             base_models.DELETION_POLICY.NOT_APPLICABLE)
 
+    def test_get_export_policy(self) -> None:
+        sample_dict = {
```

Comment:
Use "expected" instead of "sample", and call it something like expected_export_policy_dict. That will make it clear that this is the expected value for the test.

Ditto elsewhere.

Reply:
Done!